packages/palette-docs/content/docs/elements/layout/CSSGrid.mdx

```diff
@@ -0,0 +1,31 @@
+---
+name: CSSGrid
+---
+
+The CSSGrid component is a layout container for content that exposes props
+related to CSS grid.
```

Comment:
Can we move the other Grid to `layout` as well so that these items live side-by-side, and add cross references in that Grid's docs to this and vice versa?